app/src/main/java/space/taran/arkrate/presentation/App.kt

```diff
@@ -42,4 +51,26 @@ class App: Application() {
             }
         }
     }
+
+    private fun initWorker() {
+        val workManager = WorkManager.getInstance(this)
+
+        val constraints = Constraints.Builder()
+            .setRequiredNetworkType(NetworkType.CONNECTED)
+            .build()
+
+        val workRequest =
+            PeriodicWorkRequest.Builder(
+                CurrencyMonitorWorker::class.java,
+                1L,
```

Comment:
Once per day, right? Maybe it's worth to check a bit more often, e.g. 2-4 times per day. Because we might change frequency of our "GitHub cache" updates. Also, alerts should be checked, preferably, right after update otherwise we'll wait for the whole update period to check alert which will be already outdated.

Reply:
Now every 8 hours.
I have no idea how to sync checkout with github repo
It seems to me that this is not so important, maybe the backend will change later